src/chains-config/metadata-consts/polkadot-consts.ts

```diff
@@ -0,0 +1,68 @@
+import { Definition } from '../../types/chains-config';
```

Comment:
File name should be camelCase for consistency. so polkadotConsts.ts . On the other hand we do folder names are kebab-case.